Library/Homebrew/download_strategy.rb

```diff
@@ -576,13 +577,14 @@ def externals
   end
 
   def fetch_repo(target, url, revision = nil, ignore_externals = false)
-    # Use "svn up" when the repository already exists locally.
+    # Use "svn update" when the repository already exists locally.
     # This saves on bandwidth and will have a similar effect to verifying the
     # cache as it will make any changes to get the right revision.
-    svncommand = target.directory? ? "up" : "checkout"
-    args = ["svn", svncommand]
-    args << url unless target.directory?
-    args << target
+    args = if target.directory?
+      ["cd", target.to_s, ";", "svn", "update"]
```

Comment:
This also needs to use `system_command!` with `chdir:`.

Reply:
That's what I meant with my last comment, I wanted to check because of the
check ```if args[0] == "update"```...